polaris-tokens/src/themes/base/color.ts

```diff
@@ -400,1106 +235,631 @@ export const color: {
+    value: colors.red[5],
   },
   'color-bg-surface-critical-active': {
-    value: colorsExperimental.red[6],
+    value: colors.red[6],
   },
   'color-bg-fill-caution': {
-    value: colorsExperimental.yellow[6],
+    value: colors.yellow[6],
   },
   'color-bg-fill-caution-secondary': {
-    value: colorsExperimental.yellow[4],
+    value: colors.yellow[4],
   },
   'color-bg-surface-caution': {
-    value: colorsExperimental.yellow[2],
+    value: colors.yellow[2],
   },
   'color-bg-surface-caution-hover': {
-    value: colorsExperimental.yellow[3],
+    value: colors.yellow[3],
   },
   'color-bg-surface-caution-active': {
-    value: colorsExperimental.yellow[4],
+    value: colors.yellow[4],
   },
   'color-bg-fill-info': {
-    value: colorsExperimental.azure[9],
+    value: colors.azure[9],
   },
   'color-bg-fill-info-secondary': {
-    value: colorsExperimental.azure[4],
+    value: colors.azure[4],
   },
   'color-bg-surface-info': {
-    value: colorsExperimental.azure[3],
+    value: colors.azure[3],
   },
   'color-bg-surface-info-hover': {
-    value: colorsExperimental.azure[4],
+    value: colors.azure[4],
   },
   'color-bg-surface-info-active': {
-    value: colorsExperimental.azure[6],
+    value: colors.azure[6],
   },
   'color-bg-fill-warning': {
-    value: colorsExperimental.orange[9],
+    value: colors.orange[9],
   },
   'color-bg-fill-warning-secondary': {
-    value: colorsExperimental.orange[7],
+    value: colors.orange[7],
   },
   'color-bg-surface-warning': {
-    value: colorsExperimental.orange[3],
+    value: colors.orange[3],
   },
   'color-bg-fill-magic': {
-    value: colorsExperimental.purple[12],
+    value: colors.purple[12],
   },
   'color-bg-fill-magic-secondary': {
-    value: colorsExperimental.purple[6],
+    value: colors.purple[6],
   },
   'color-bg-fill-magic-secondary-hover': {
-    value: colorsExperimental.purple[7],
+    value: colors.purple[7],
   },
   'color-bg-fill-magic-secondary-active': {
-    value: colorsExperimental.purple[8],
+    value: colors.purple[8],
   },
   'color-bg-surface-magic-hover': {
-    value: colorsExperimental.purple[4],
+    value: colors.purple[4],
   },
   'color-bg-fill-secondary': {
-    value: colorsExperimental.gray[6],
+    value: colors.gray[6],
   },
   'color-bg-fill-inverse': {
-    value: colorsExperimental.gray[15],
+    value: colors.gray[15],
   },
   'color-bg-fill-inverse-hover': {
-    value: colorsExperimental.gray[14],
+    value: colors.gray[14],
   },
   'color-bg-fill-inverse-active': {
-    value: colorsExperimental.gray[13],
+    value: colors.gray[13],
   },
   'color-bg-surface-transparent': {
-    value: colorsExperimental.blackAlpha[1],
+    value: colors.blackAlpha[1],
   },
   'color-bg-fill-transparent-hover': {
-    value: colorsExperimental.blackAlpha[5],
+    value: colors.blackAlpha[5],
   },
   'color-bg-fill-transparent-active': {
-    value: colorsExperimental.blackAlpha[6],
+    value: colors.blackAlpha[6],
```

Comment:
```suggestion
    value: colors.blackAlpha[7],
```
This is listed as [7 in Figma](https://www.figma.com/file/oKq2CLPYeX5qK5frzwipM3/Polaris-Gen-3-Styles?type=design&node-id=14696%3A1999&mode=design&t=vGyfRzAa3mogdM3j-1). We need to determine which of these values is correct.

<img width="509" alt="Screenshot 2023-10-02 at 11 28 44 AM" src="https://github.com/Shopify/polaris/assets/21976492/1897f318-6e6d-456b-a6cb-30f231adb59f">